tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -21,3 +30,45 @@ def __init__(self, marionette):
     def select_time_format(self, time_format):
         self.marionette.find_element(*self._24h_selector_locator).tap()
         self.select(time_format)
+
+    def change_automatic_time_update(self, state):
+
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(
+                expected.element_present(*self._autotime_enabled_locator))))
+
+        self.marionette.find_element(*self._autotime_enabled_switch_locator).tap()
+
+        if not state:
+            self.wait_for_condition(lambda m: self.is_autotime_enabled is False)
+        else:
+            self.wait_for_condition(lambda m: self.is_autotime_enabled is True)
+
```

Comment:
Why not call this function toggle_automatic_time_update() and then check that at the end the old self.is_autotime_enabled != new self.is_autotime_enabled?
That way, you don't need a function argument at all. During the test, you should know the state of self.is_autotime_enabled anyway. You can do that check in the test file itself.